main.py

```diff
@@ -0,0 +1,49 @@
+import random
+
+def input_name():
+    """
+    Asks the user to input a list of names, validates them, removes duplicates and returns a list of valid names.
+    """
+    # Ask the user to input a string of names separated by spaces
+    name_input = input("Please enter alpha names separated by spaces: ")
```

Comment:
Perhaps leave comments in python style comments, comment about the code above or below rather than on the same line